### ISSUE TITLE
Artist only view

### ISSUE THREAD

**Issue by thiagobucca** [Sonerezh/sonerezh#330]
Hello,

Is there a way to display an artist only view? without showing each album for each artist?

**Comment by lGuillaume124** [comment]
Hello,

Not yet, this is due to a bad database design.

**Comment by techexo** [comment]
Yes, that would be a **huge** benefit, because it's currently a pain if we want to see everything done by "Zaz", for example 😄 . You have to scroll down all the way through all the albums before being able to find the artist!

I understand that's why the search feature exits, but it seems weird not to have an artist view with only the artists.
